contrib/kind.sh

```diff
@@ -204,6 +206,9 @@ parse_args() {
                                                 ;;
             -df | --disable-forwarding )        OVN_DISABLE_FORWARDING=true
                                                 ;;
+            -ecp | --encap-port )               shift
```

Comment:
do we want to support this as u/s user facing CLI? we should update the kind.md docs with the new option

Reply:
I think that documenting the options in kind.md via just printing out the same thing that `kind.sh --help` prints out is just a waste of time. It is already outdated and honestly a nuisance to keep in sync. I think we should just refer there to the possibility of using `kind.sh --help` where most options are self explanatory and just provide some deeper explanation on more complex configuration options. I can clean that up in a follow up PR. WDYT?

Reply:
ack i am ok with deleting that doc if that's what we want, but deff that wasn't stale.. i think most of them were updating it with the new options.
so +1 to either removing it OR updating it. Stale docs are yuck.